scripts/_skeleton_role_/meta/main.yml.j2

```diff
@@ -10,5 +10,12 @@ galaxy_info:
       versions:
         - 30
   galaxy_tags: ["os-migrate"]
+{% if 'export' in role_name %}
```

Comment:
Both conditionals could be done in this if statement to deduplicate a little.

Reply:
There's 'prelude_src' vs. 'prelude_dst' difference in the dependency in each conditional though, so IMO it's good?

Reply:
yup, they are different, I had the 'dependencies:' outside, but then I remembered that we might have different roles without any dependency

Reply:
Oh duh. OK never mind. 👍